### ISSUE TITLE
Encoding in iptc fields

### ISSUE THREAD

**Issue by uasd7** [mattiasw/ExifReader#36]
Hello Mattias,
I have a similar problem as in [https://github.com/mattiasw/ExifReader/issues/26](https://github.com/mattiasw/ExifReader/issues/26) . Special characters in certain fields are not displayed correctly.  But it works with other fields. I made a dirty hack for myself: I run all fields through decodeURIComponent(escape(.....)). That's not a good idea, but it fixes my current problem. Do you have any idea why the encoding goes wrong? Using PHP with iptcparse I get correct values.
This is an example image: [http://www.bildbetextung.de/img/slider/iptc_sample_13.jpg](http://www.bildbetextung.de/img/slider/iptc_sample_13.jpg)

Here is what i get with "expanded: true" (search for "ttingen") :
```javascript
{
    "exif": {
        "ImageWidth": {
            "value": 3543,
            "description": 3543
        },
        "ImageLength": {
            "value": 2362,
            "description": 2362
        },
        "BitsPerSample": {
            "value": [8, 8, 8],
            "description": "8, 8, 8"
        },
        "PhotometricInterpretation": {
            "value": 2,
            "description": 2
        },
        "ImageDescription": {
            "value": ["Planetenweg Göttingen Station Sonne Start des Weges in der  Goethalle Der Planetenweg ist eine populärastronomische Installation im öffentlichen Raum in Göttingen. Der Planetenweg besteht aus aus zehn Bronzestelen, die die Sonne, die acht \"normalen\" Planenten unseres Sonnensystems sowie Pluto zeigen. Auf den Bronzestelen finden sich jeweils Informationen zu den Planeten. Der Göttinger Planentweg ist im Maßstab 1:2 Milliarden gehalten. Zwei Millarden Kilometer im Weltall entsprechen damit einem Kilometer auf dem Planetenweg. Die Erde schrumpft auf eine 6,5 Millimeter kleine Kugel; die Sonne ist dagegen 70 Zentimeter groß.\nDer Planetenweg erstreckt sich von der Sonnen-Stele in der Goethe-Allee bis zum Bismarckturm, wo die Pluto-Stele zu finden ist. Pluto wurde am 26. August 2006 der Status als Planet aberkannt; Pluto ist seitdem nur noch ein \"Zwergplanet\".\nDer Planetenweg wurde initiiert vom Förderverein Planetarium Göttingen e.V.. Die Bronze-Stelen des Planentenwegs wurden durch Sponsorengelder finanziert und maßgeblich von Reinhold Wittig gestaltet. (Quelle: Wiki Göttingen)\n"],
            "description": "Planetenweg Göttingen Station Sonne Start des Weges in der  Goethalle Der Planetenweg ist eine populärastronomische Installation im öffentlichen Raum in Göttingen. Der Planetenweg besteht aus aus zehn Bronzestelen, die die Sonne, die acht \"normalen\" Planenten unseres Sonnensystems sowie Pluto zeigen. Auf den Bronzestelen finden sich jeweils Informationen zu den Planeten. Der Göttinger Planentweg ist im Maßstab 1:2 Milliarden gehalten. Zwei Millarden Kilometer im Weltall entsprechen damit einem Kilometer auf dem Planetenweg. Die Erde schrumpft auf eine 6,5 Millimeter kleine Kugel; die Sonne ist dagegen 70 Zentimeter groß.\nDer Planetenweg erstreckt sich von der Sonnen-Stele in der Goethe-Allee bis zum Bismarckturm, wo die Pluto-Stele zu finden ist. Pluto wurde am 26. August 2006 der Status als Planet aberkannt; Pluto ist seitdem nur noch ein \"Zwergplanet\".\nDer Planetenweg wurde initiiert vom Förderverein Planetarium Göttingen e.V.. Die Bronze-Stelen des Planentenwegs wurden durch Sponsorengelder finanziert und maßgeblich von Reinhold Wittig gestaltet. (Quelle: Wiki Göttingen)\n"
        },
        "Orientation": {
            "value": 1,
            "description": "top-left"
        },
        "SamplesPerPixel": {
            "value": 3,
            "description": 3
        },
        "XResolution": {
            "value": 300,
            "description": 300
        },
        "YResolution": {
            "value": 300,
            "description": 300
        },
        "ResolutionUnit": {
            "value": 2,
            "description": "inches"
        },
        "Software": {
            "value": ["Adobe Photoshop CS5 Macintosh"],
            "description": "Adobe Photoshop CS5 Macintosh"
        },
        "DateTime": {
            "value": ["2012:08:29 14:51:10"],
            "description": "2012:08:29 14:51:10"
        },
        "Artist": {
            "value": ["Bernd Beuermann"],
            "description": "Bernd Beuermann"
        },
        "Copyright": {
            "value": ["Bernd Beuermann"],
            "description": "Bernd Beuermann"
        },
        "Exif IFD Pointer": {
            "value": 1404,
            "description": 1404
        },
        "ExifVersion": {
            "value": [48, 50, 50, 49],
            "description": "0221"
        },
        "ColorSpace": {
            "value": 65535,
            "description": "Uncalibrated"
        },
        "PixelXDimension": {
            "value": 694,
            "description": 694
        },
        "PixelYDimension": {
            "value": 340,
            "description": 340
        }
    },
    "iptc": {
        "Coded Character Set": {
            "value": [27, 37, 71],
            "description": "UTF-8"
        },
        "Record Version": {
            "value": [0, 3],
            "description": "3"
        },
        "Caption/Abstract": {
            "value": [80, 108, 97, 110, 101, 116, 101, 110, 119, 101, 103, 32, 71, 195, 182, 116, 116, 105, 110, 103, 101, 110, 32, 83, 116, 97, 116, 105, 111, 110, 32, 83, 111, 110, 110, 101, 32, 83, 116, 97, 114, 116, 32, 100, 101, 115, 32, 87, 101, 103, 101, 115, 32, 105, 110, 32, 100, 101, 114, 32, 32, 71, 111, 101, 116, 104, 97, 108, 108, 101, 32, 68, 101, 114, 32, 80, 108, 97, 110, 101, 116, 101, 110, 119, 101, 103, 32, 105, 115, 116, 32, 101, 105, 110, 101, 32, 112, 111, 112, 117, 108, 195, 164, 114, 97, 115, 116, 114, 111, 110, 111, 109, 105, 115, 99, 104, 101, 32, 73, 110, 115, 116, 97, 108, 108, 97, 116, 105, 111, 110, 32, 105, 109, 32, 195, 182, 102, 102, 101, 110, 116, 108, 105, 99, 104, 101, 110, 32, 82, 97, 117, 109, 32, 105, 110, 32, 71, 195, 182, 116, 116, 105, 110, 103, 101, 110, 46, 32, 68, 101, 114, 32, 80, 108, 97, 110, 101, 116, 101, 110, 119, 101, 103, 32, 98, 101, 115, 116, 101, 104, 116, 32, 97, 117, 115, 32, 97, 117, 115, 32, 122, 101, 104, 110, 32, 66, 114, 111, 110, 122, 101, 115, 116, 101, 108, 101, 110, 44, 32, 100, 105, 101, 32, 100, 105, 101, 32, 83, 111, 110, 110, 101, 44, 32, 100, 105, 101, 32, 97, 99, 104, 116, 32, 34, 110, 111, 114, 109, 97, 108, 101, 110, 34, 32, 80, 108, 97, 110, 101, 110, 116, 101, 110, 32, 117, 110, 115, 101, 114, 101, 115, 32, 83, 111, 110, 110, 101, 110, 115, 121, 115, 116, 101, 109, 115, 32, 115, 111, 119, 105, 101, 32, 80, 108, 117, 116, 111, 32, 122, 101, 105, 103, 101, 110, 46, 32, 65, 117, 102, 32, 100, 101, 110, 32, 66, 114, 111, 110, 122, 101, 115, 116, 101, 108, 101, 110, 32, 102, 105, 110, 100, 101, 110, 32, 115, 105, 99, 104, 32, 106, 101, 119, 101, 105, 108, 115, 32, 73, 110, 102, 111, 114, 109, 97, 116, 105, 111, 110, 101, 110, 32, 122, 117, 32, 100, 101, 110, 32, 80, 108, 97, 110, 101, 116, 101, 110, 46, 32, 68, 101, 114, 32, 71, 195, 182, 116, 116, 105, 110, 103, 101, 114, 32, 80, 108, 97, 110, 101, 110, 116, 119, 101, 103, 32, 105, 115, 116, 32, 105, 109, 32, 77, 97, 195, 159, 115, 116, 97, 98, 32, 49, 58, 50, 32, 77, 105, 108, 108, 105, 97, 114, 100, 101, 110, 32, 103, 101, 104, 97, 108, 116, 101, 110, 46, 32, 90, 119, 101, 105, 32, 77, 105, 108, 108, 97, 114, 100, 101, 110, 32, 75, 105, 108, 111, 109, 101, 116, 101, 114, 32, 105, 109, 32, 87, 101, 108, 116, 97, 108, 108, 32, 101, 110, 116, 115, 112, 114, 101, 99, 104, 101, 110, 32, 100, 97, 109, 105, 116, 32, 101, 105, 110, 101, 109, 32, 75, 105, 108, 111, 109, 101, 116, 101, 114, 32, 97, 117, 102, 32, 100, 101, 109, 32, 80, 108, 97, 110, 101, 116, 101, 110, 119, 101, 103, 46, 32, 68, 105, 101, 32, 69, 114, 100, 101, 32, 115, 99, 104, 114, 117, 109, 112, 102, 116, 32, 97, 117, 102, 32, 101, 105, 110, 101, 32, 54, 44, 53, 32, 77, 105, 108, 108, 105, 109, 101, 116, 101, 114, 32, 107, 108, 101, 105, 110, 101, 32, 75, 117, 103, 101, 108, 59, 32, 100, 105, 101, 32, 83, 111, 110, 110, 101, 32, 105, 115, 116, 32, 100, 97, 103, 101, 103, 101, 110, 32, 55, 48, 32, 90, 101, 110, 116, 105, 109, 101, 116, 101, 114, 32, 103, 114, 111, 195, 159, 46, 13, 68, 101, 114, 32, 80, 108, 97, 110, 101, 116, 101, 110, 119, 101, 103, 32, 101, 114, 115, 116, 114, 101, 99, 107, 116, 32, 115, 105, 99, 104, 32, 118, 111, 110, 32, 100, 101, 114, 32, 83, 111, 110, 110, 101, 110, 45, 83, 116, 101, 108, 101, 32, 105, 110, 32, 100, 101, 114, 32, 71, 111, 101, 116, 104, 101, 45, 65, 108, 108, 101, 101, 32, 98, 105, 115, 32, 122, 117, 109, 32, 66, 105, 115, 109, 97, 114, 99, 107, 116, 117, 114, 109, 44, 32, 119, 111, 32, 100, 105, 101, 32, 80, 108, 117, 116, 111, 45, 83, 116, 101, 108, 101, 32, 122, 117, 32, 102, 105, 110, 100, 101, 110, 32, 105, 115, 116, 46, 32, 80, 108, 117, 116, 111, 32, 119, 117, 114, 100, 101, 32, 97, 109, 32, 50, 54, 46, 32, 65, 117, 103, 117, 115, 116, 32, 50, 48, 48, 54, 32, 100, 101, 114, 32, 83, 116, 97, 116, 117, 115, 32, 97, 108, 115, 32, 80, 108, 97, 110, 101, 116, 32, 97, 98, 101, 114, 107, 97, 110, 110, 116, 59, 32, 80, 108, 117, 116, 111, 32, 105, 115, 116, 32, 115, 101, 105, 116, 100, 101, 109, 32, 110, 117, 114, 32, 110, 111, 99, 104, 32, 101, 105, 110, 32, 34, 90, 119, 101, 114, 103, 112, 108, 97, 110, 101, 116, 34, 46, 13, 68, 101, 114, 32, 80, 108, 97, 110, 101, 116, 101, 110, 119, 101, 103, 32, 119, 117, 114, 100, 101, 32, 105, 110, 105, 116, 105, 105, 101, 114, 116, 32, 118, 111, 109, 32, 70, 195, 182, 114, 100, 101, 114, 118, 101, 114, 101, 105, 110, 32, 80, 108, 97, 110, 101, 116, 97, 114, 105, 117, 109, 32, 71, 195, 182, 116, 116, 105, 110, 103, 101, 110, 32, 101, 46, 86, 46, 46, 32, 68, 105, 101, 32, 66, 114, 111, 110, 122, 101, 45, 83, 116, 101, 108, 101, 110, 32, 100, 101, 115, 32, 80, 108, 97, 110, 101, 110, 116, 101, 110, 119, 101, 103, 115, 32, 119, 117, 114, 100, 101, 110, 32, 100, 117, 114, 99, 104, 32, 83, 112, 111, 110, 115, 111, 114, 101, 110, 103, 101, 108, 100, 101, 114, 32, 102, 105, 110, 97, 110, 122, 105, 101, 114, 116, 32, 117, 110, 100, 32, 109, 97, 195, 159, 103, 101, 98, 108, 105, 99, 104, 32, 118, 111, 110, 32, 82, 101, 105, 110, 104, 111, 108, 100, 32, 87, 105, 116, 116, 105, 103, 32, 103, 101, 115, 116, 97, 108, 116, 101, 116, 46, 32, 40, 81, 117, 101, 108, 108, 101, 58, 32, 87, 105, 107, 105, 32, 71, 195, 182, 116, 116, 105, 110, 103, 101, 110, 41, 13],
            "description": "Planetenweg GÃ¶ttingen Station Sonne Start des Weges in der  Goethalle Der Planetenweg ist eine populÃ¤rastronomische Installation im Ã¶ffentlichen Raum in GÃ¶ttingen. Der Planetenweg besteht aus aus zehn Bronzestelen, die die Sonne, die acht \"normalen\" Planenten unseres Sonnensystems sowie Pluto zeigen. Auf den Bronzestelen finden sich jeweils Informationen zu den Planeten. Der GÃ¶ttinger Planentweg ist im MaÃstab 1:2 Milliarden gehalten. Zwei Millarden Kilometer im Weltall entsprechen damit einem Kilometer auf dem Planetenweg. Die Erde schrumpft auf eine 6,5 Millimeter kleine Kugel; die Sonne ist dagegen 70 Zentimeter groÃ.\rDer Planetenweg erstreckt sich von der Sonnen-Stele in der Goethe-Allee bis zum Bismarckturm, wo die Pluto-Stele zu finden ist. Pluto wurde am 26. August 2006 der Status als Planet aberkannt; Pluto ist seitdem nur noch ein \"Zwergplanet\".\rDer Planetenweg wurde initiiert vom FÃ¶rderverein Planetarium GÃ¶ttingen e.V.. Die Bronze-Stelen des Planentenwegs wurden durch Sponsorengelder finanziert und maÃgeblich von Reinhold Wittig gestaltet. (Quelle: Wiki GÃ¶ttingen)\r"
        },
        "Headline": {
            "value": [80, 108, 97, 110, 101, 116, 101, 110, 119, 101, 103, 32, 71, 195, 182, 116, 116, 105, 110, 103, 101, 110, 32, 83, 116, 97, 114, 116, 32, 83, 116, 97, 116, 105, 111, 110, 32, 83, 111, 110, 110, 101],
            "description": "Planetenweg GÃ¶ttingen Start Station Sonne"
        },
        "By-line": {
            "value": [66, 101, 114, 110, 100, 32, 66, 101, 117, 101, 114, 109, 97, 110, 110],
            "description": "Bernd Beuermann"
        },
        "Credit": {
            "value": [101, 117, 114, 111, 112, 105, 120, 101, 108, 47, 66, 46, 66, 101, 117, 101, 114, 109, 97, 110, 110],
            "description": "europixel/B.Beuermann"
        },
        "Source": {
            "value": [101, 117, 114, 111, 112, 105, 120, 101, 108, 46, 101, 117],
            "description": "europixel.eu"
        },
        "Object Name": {
            "value": [80, 108, 97, 110, 101, 116, 101, 110, 119, 101, 103, 32, 71, 195, 182, 116, 116, 105, 110, 103, 101, 110, 32, 83, 116, 97, 114, 116, 32, 83, 116, 97, 116, 105, 111, 110, 32, 83, 111, 110, 110, 101],
            "description": "Planetenweg GÃ¶ttingen Start Station Sonne"
        },
        "Date Created": {
            "value": [50, 48, 49, 50, 48, 56, 49, 55],
            "description": "2012-08-17"
        },
        "Time Created": {
            "value": [48, 55, 48, 52, 52, 53, 43, 48, 49, 48, 48],
            "description": "07:04:45+01:00"
        },
        "City": {
            "value": [71, 195, 182, 116, 116, 105, 110, 103, 101, 110],
            "description": "GÃ¶ttingen"
        },
        "Sub-location": {
            "value": [71, 111, 101, 116, 104, 101, 45, 65, 108, 108, 101, 101],
            "description": "Goethe-Allee"
        },
        "Province/State": {
            "value": [78, 105, 101, 100, 101, 114, 115, 97, 99, 104, 115, 101, 110],
            "description": "Niedersachsen"
        },
        "Country/Primary Location Name": {
            "value": [68, 101, 117, 116, 115, 99, 104, 108, 97, 110, 100],
            "description": "Deutschland"
        },
        "Country/Primary Location Code": {
            "value": [68, 69, 85],
            "description": "DEU"
        },
        "Keywords": [{
            "value": [80, 108, 97, 110, 101, 116, 101, 110, 119, 101, 103],
            "description": "Planetenweg"
        }, {
            "value": [65, 115, 116, 114, 111, 110, 111, 109, 105, 101],
            "description": "Astronomie"
        }, {
            "value": [80, 108, 97, 110, 101, 116, 101, 110],
            "description": "Planeten"
        }, {
            "value": [71, 195, 182, 116, 116, 105, 110, 103, 101, 110],
            "description": "GÃ¶ttingen"
        }, {
            "value": [87, 101, 103],
            "description": "Weg"
        }, {
            "value": [87, 105, 115, 115, 101, 110],
            "description": "Wissen"
        }, {
            "value": [66, 105, 108, 100, 117, 110, 103],
            "description": "Bildung"
        }, {
            "value": [70, 114, 101, 105, 122, 101, 105, 116],
            "description": "Freizeit"
        }],
        "Copyright Notice": {
            "value": [66, 101, 114, 110, 100, 32, 66, 101, 117, 101, 114, 109, 97, 110, 110],
            "description": "Bernd Beuermann"
        },
        "undefined-733": {
            "value": [48, 58, 48, 58, 48, 58, 45, 48, 48, 48, 48, 49],
            "description": [48, 58, 48, 58, 48, 58, 45, 48, 48, 48, 48, 49]
        }
    },
    "xmp": {
        "about": {
            "value": "",
            "attributes": {},
            "description": ""
        },
        "ModifyDate": {
            "value": "2012-08-29T14:51:10+02:00",
            "attributes": {},
            "description": "2012-08-29T14:51:10+02:00"
        },
        "MetadataDate": {
            "value": "2012-08-29T14:51:10+02:00",
            "attributes": {},
            "description": "2012-08-29T14:51:10+02:00"
        },
        "Label": {
            "value": "",
            "attributes": {},
            "description": ""
        },
        "Rating": {
            "value": "100",
            "attributes": {},
            "description": "100"
        },
        "CreateDate": {
            "value": "2012-08-16T12:57:16+02:00",
            "attributes": {},
            "description": "2012-08-16T12:57:16+02:00"
        },
        "format": {
            "value": "image/jpeg",
            "attributes": {},
            "description": "image/jpeg"
        },
        "City": {
            "value": "GÃ¶ttingen",
            "attributes": {},
            "description": "Göttingen"
        },
        "State": {
            "value": "Niedersachsen",
            "attributes": {},
            "description": "Niedersachsen"
        },
        "Country": {
            "value": "Deutschland",
            "attributes": {},
            "description": "Deutschland"
        },
        "Credit": {
            "value": "europixel/B.Beuermann",
            "attributes": {},
            "description": "europixel/B.Beuermann"
        },
        "Source": {
            "value": "europixel.eu",
            "attributes": {},
            "description": "europixel.eu"
        },
        "Headline": {
            "value": "Planetenweg GÃ¶ttingen Start Station Sonne",
            "attributes": {},
            "description": "Planetenweg Göttingen Start Station Sonne"
        },
        "DateCreated": {
            "value": "2012-08-17T07:04:45+01:00",
            "attributes": {},
            "description": "2012-08-17T07:04:45+01:00"
        },
        "LegacyIPTCDigest": {
            "value": "0E6815C71DFE818EE54C713049899D30",
            "attributes": {},
            "description": "0E6815C71DFE818EE54C713049899D30"
        },
        "ColorMode": {
            "value": "3",
            "attributes": {},
            "description": "3"
        },
        "InstanceID": {
            "value": "xmp.iid:0A127F350920681197A5A601B1FCC622",
            "attributes": {},
            "description": "xmp.iid:0A127F350920681197A5A601B1FCC622"
        },
        "DocumentID": {
            "value": "xmp.did:CA4D19C70A206811994C88E52234E22C",
            "attributes": {},
            "description": "xmp.did:CA4D19C70A206811994C88E52234E22C"
        },
        "OriginalDocumentID": {
            "value": "xmp.did:CA4D19C70A206811994C88E52234E22C",
            "attributes": {},
            "description": "xmp.did:CA4D19C70A206811994C88E52234E22C"
        },
        "CountryCode": {
            "value": "DEU",
            "attributes": {},
            "description": "DEU"
        },
        "Location": {
            "value": "Goethe-Allee",
            "attributes": {},
            "description": "Goethe-Allee"
        },
        "Marked": {
            "value": "True",
            "attributes": {},
            "description": "True"
        },
        "ColorClass": {
            "value": "0",
            "attributes": {},
            "description": "0"
        },
        "Tagged": {
            "value": "False",
            "attributes": {},
            "description": "False"
        },
        "Prefs": {
            "value": "0:0:0:-00001",
            "attributes": {},
            "description": "0:0:0:-00001"
        },
        "PMVersion": {
            "value": "PM5",
            "attributes": {},
            "description": "PM5"
        },
        "AlreadyApplied": {
            "value": "True",
            "attributes": {},
            "description": "True"
        },
        "creator": {
            "value": [{
                "value": "Bernd Beuermann",
                "attributes": {},
                "description": "Bernd Beuermann"
            }],
            "attributes": {},
            "description": "Bernd Beuermann"
        },
        "title": {
            "value": [{
                "value": "Planetenweg GÃ¶ttingen Start Station Sonne",
                "attributes": {
                    "lang": "x-default"
                },
                "description": "Planetenweg Göttingen Start Station Sonne"
            }],
            "attributes": {},
            "description": "Planetenweg Göttingen Start Station Sonne"
        },
        "subject": {
            "value": [{
                "value": "Planetenweg",
                "attributes": {},
                "description": "Planetenweg"
            }, {
                "value": "Astronomie",
                "attributes": {},
                "description": "Astronomie"
            }, {
                "value": "Planeten",
                "attributes": {},
                "description": "Planeten"
            }, {
                "value": "GÃ¶ttingen",
                "attributes": {},
                "description": "Göttingen"
            }, {
                "value": "Weg",
                "attributes": {},
                "description": "Weg"
            }, {
                "value": "Wissen",
                "attributes": {},
                "description": "Wissen"
            }, {
                "value": "Bildung",
                "attributes": {},
                "description": "Bildung"
            }, {
                "value": "Freizeit",
                "attributes": {},
                "description": "Freizeit"
            }],
            "attributes": {},
            "description": "Planetenweg, Astronomie, Planeten, Göttingen, Weg, Wissen, Bildung, Freizeit"
        },
        "rights": {
            "value": [{
                "value": "Bernd Beuermann",
                "attributes": {
                    "lang": "x-default"
                },
                "description": "Bernd Beuermann"
            }],
            "attributes": {},
            "description": "Bernd Beuermann"
        },
        "description": {
            "value": [{
                "value": "Planetenweg GÃ¶ttingen Station Sonne Start des Weges in der  Goethalle Der Planetenweg ist eine populÃ¤rastronomische Installation im Ã¶ffentlichen Raum in GÃ¶ttingen. Der Planetenweg besteht aus aus zehn Bronzestelen, die die Sonne, die acht \"normalen\" Planenten unseres Sonnensystems sowie Pluto zeigen. Auf den Bronzestelen finden sich jeweils Informationen zu den Planeten. Der GÃ¶ttinger Planentweg ist im MaÃstab 1:2 Milliarden gehalten. Zwei Millarden Kilometer im Weltall entsprechen damit einem Kilometer auf dem Planetenweg. Die Erde schrumpft auf eine 6,5 Millimeter kleine Kugel; die Sonne ist dagegen 70 Zentimeter groÃ.\nDer Planetenweg erstreckt sich von der Sonnen-Stele in der Goethe-Allee bis zum Bismarckturm, wo die Pluto-Stele zu finden ist. Pluto wurde am 26. August 2006 der Status als Planet aberkannt; Pluto ist seitdem nur noch ein \"Zwergplanet\".\nDer Planetenweg wurde initiiert vom FÃ¶rderverein Planetarium GÃ¶ttingen e.V.. Die Bronze-Stelen des Planentenwegs wurden durch Sponsorengelder finanziert und maÃgeblich von Reinhold Wittig gestaltet. (Quelle: Wiki GÃ¶ttingen)\n",
                "attributes": {
                    "lang": "x-default"
                },
                "description": "Planetenweg Göttingen Station Sonne Start des Weges in der  Goethalle Der Planetenweg ist eine populärastronomische Installation im öffentlichen Raum in Göttingen. Der Planetenweg besteht aus aus zehn Bronzestelen, die die Sonne, die acht \"normalen\" Planenten unseres Sonnensystems sowie Pluto zeigen. Auf den Bronzestelen finden sich jeweils Informationen zu den Planeten. Der Göttinger Planentweg ist im Maßstab 1:2 Milliarden gehalten. Zwei Millarden Kilometer im Weltall entsprechen damit einem Kilometer auf dem Planetenweg. Die Erde schrumpft auf eine 6,5 Millimeter kleine Kugel; die Sonne ist dagegen 70 Zentimeter groß.\nDer Planetenweg erstreckt sich von der Sonnen-Stele in der Goethe-Allee bis zum Bismarckturm, wo die Pluto-Stele zu finden ist. Pluto wurde am 26. August 2006 der Status als Planet aberkannt; Pluto ist seitdem nur noch ein \"Zwergplanet\".\nDer Planetenweg wurde initiiert vom Förderverein Planetarium Göttingen e.V.. Die Bronze-Stelen des Planentenwegs wurden durch Sponsorengelder finanziert und maßgeblich von Reinhold Wittig gestaltet. (Quelle: Wiki Göttingen)\n"
            }],
            "attributes": {},
            "description": "Planetenweg Göttingen Station Sonne Start des Weges in der  Goethalle Der Planetenweg ist eine populärastronomische Installation im öffentlichen Raum in Göttingen. Der Planetenweg besteht aus aus zehn Bronzestelen, die die Sonne, die acht \"normalen\" Planenten unseres Sonnensystems sowie Pluto zeigen. Auf den Bronzestelen finden sich jeweils Informationen zu den Planeten. Der Göttinger Planentweg ist im Maßstab 1:2 Milliarden gehalten. Zwei Millarden Kilometer im Weltall entsprechen damit einem Kilometer auf dem Planetenweg. Die Erde schrumpft auf eine 6,5 Millimeter kleine Kugel; die Sonne ist dagegen 70 Zentimeter groß.\nDer Planetenweg erstreckt sich von der Sonnen-Stele in der Goethe-Allee bis zum Bismarckturm, wo die Pluto-Stele zu finden ist. Pluto wurde am 26. August 2006 der Status als Planet aberkannt; Pluto ist seitdem nur noch ein \"Zwergplanet\".\nDer Planetenweg wurde initiiert vom Förderverein Planetarium Göttingen e.V.. Die Bronze-Stelen des Planentenwegs wurden durch Sponsorengelder finanziert und maßgeblich von Reinhold Wittig gestaltet. (Quelle: Wiki Göttingen)\n"
        },
        "History": {
            "value": [{
                "value": {},
                "attributes": {
                    "action": "saved",
                    "instanceID": "xmp.iid:CA4D19C70A206811994C88E52234E22C",
                    "when": "2012-08-16T15:17:58+02:00",
                    "softwareAgent": "Adobe Photoshop CS5 Macintosh",
                    "changed": "/"
                },
                "description": ""
            }, {
                "value": {},
                "attributes": {
                    "action": "saved",
                    "instanceID": "xmp.iid:09127F350920681197A5A601B1FCC622",
                    "when": "2012-08-29T14:51:10+02:00",
                    "softwareAgent": "Adobe Photoshop CS5 Macintosh",
                    "changed": "/"
                },
                "description": ""
            }, {
                "value": {},
                "attributes": {
                    "action": "saved",
                    "instanceID": "xmp.iid:0A127F350920681197A5A601B1FCC622",
                    "when": "2012-08-29T14:51:10+02:00",
                    "softwareAgent": "Adobe Photoshop CS5 Macintosh",
                    "changed": "/"
                },
                "description": ""
            }],
            "attributes": {},
            "description": ", , "
        },
        "CreatorContactInfo": {
            "value": {
                "CiAdrExtadr": {
                    "value": "Bernd Beuermann, Garbenstr. 32, 37081 GÃ¶ttingen",
                    "attributes": {},
                    "description": "Bernd Beuermann, Garbenstr. 32, 37081 Göttingen"
                },
                "CiAdrCity": {
                    "value": "GÃ¶ttingen",
                    "attributes": {},
                    "description": "Göttingen"
                },
                "CiAdrRegion": {
                    "value": "Niedersachsen",
                    "attributes": {},
                    "description": "Niedersachsen"
                },
                "CiAdrPcode": {
                    "value": "D-37081",
                    "attributes": {},
                    "description": "D-37081"
                },
                "CiAdrCtry": {
                    "value": "Deutschland",
                    "attributes": {},
                    "description": "Deutschland"
                },
                "CiEmailWork": {
                    "value": "hello@beuermann.com",
                    "attributes": {},
                    "description": "hello@beuermann.com"
                },
                "CiUrlWork": {
                    "value": "www.beuermann.de",
                    "attributes": {},
                    "description": "www.beuermann.de"
                }
            },
            "attributes": {},
            "description": "CreatorAddress: Bernd Beuermann, Garbenstr. 32, 37081 GÃ¶ttingen; CreatorCity: GÃ¶ttingen; CreatorRegion: Niedersachsen; CreatorPostalCode: D-37081; CreatorCountry: Deutschland; CreatorWorkEmail: hello@beuermann.com; CreatorWorkUrl: www.beuermann.de"
        }
    }
}
```

**Comment by mattiasw** [comment]
Thanks for reporting. I'll try to fix it as soon as I can. I think it's not a big change.

**Comment by uasd7** [comment]
Ah perfect.
And thank you very much for your effort with this library.

**Comment by mattiasw** [comment]
Please try this fix out: #37
It actually fixed the encodings in several of my test images. :-)

**Comment by mattiasw** [comment]
Please try this fix out: #37
It actually fixed the encodings in several of my test images. :-)

**Comment by uasd7** [comment]
Mmh, you wrote almost exactly the same code as me and it works perfectly. But escape is deprecated, isn't it? You think it's a good idea to put that in?

**Comment by mattiasw** [comment]
Oh, I didn't realize that. Thanks for pointing it out. I'm not sure how to solve it then though. `escape` is already used for regular and XMP tags so I'm thinking about maybe putting it in for IPTC tags also for now (better than nothing I guess) and making a note for myself to replace it with something else (maybe `TextDecoder` could work, but that doesn't yet have the same support as `encode`). What do you think?

**Comment by uasd7** [comment]
If you already have it in the project anyway, there is no reason not to build the fix this way. 👍

**Comment by mattiasw** [comment]
:+1: 
Closed by #37.
